tests/aggregation/tests.py

```diff
@@ -497,13 +497,13 @@ def test_aggregate_annotation(self):
 
     def test_avg_duration_field(self):
         self.assertEqual(
-            Publisher.objects.aggregate(Avg('duration', output_field=DurationField())),
+            Publisher.objects.aggregate(Avg('duration')),
             {'duration__avg': datetime.timedelta(days=1, hours=12)}
         )
```

Comment:
I could keep this explicit `output_field` version and add an explicit below if required.